app/src/common/hooks/useFhirResourceTreeData.ts

```diff
@@ -196,24 +208,26 @@ const buildTree = (
     (!previousElementNode.isArray || rootNode === previousElementNode)
   ) {
     previousElementNode.children.push(currentElementNode);
-    buildTree(rest, rootNode, currentElementNode);
+    buildTree(rest, rootNode, currentElementNode, parentPath);
   } else {
     const parent = getParent(previousElementNode, rootNode);
-    if (parent) buildTree(elementsDefinition, rootNode, parent);
+    if (parent) buildTree(elementsDefinition, rootNode, parent, parentPath);
```

Comment:
is parentPath the same as parent.path ?

Reply:
Not exactly, but i'm wondering if it would not be the same as `rootNode.path`